test/bugs-reported/bug6.chpl

```diff
@@ -0,0 +1,14 @@
+use Queue;
```

Comment:
I believe this is an error because the compiler fails to find a file named "Queue.chpl". @lydia-duncan , can you chime in here? Should this be a feature request in JIRA instead?